test/motion.test.ts

```diff
@@ -120,13 +120,12 @@ suite("motion", () => {
     suite("line up", () => {
         motionModes.forEach(o => {
             test("should move cursor one line up", () => {
+
                 let motion = new Motion(o).move(1, 0);
                 assert.equal(motion.position.line, 1);
-                assert.equal(motion.position.character, 0);
```

Comment:
The test only mentions the line. Plus the _desiredColumn value is cached and the test failed. It probably needs to have the _desiredColumn reset beforeEach